supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,11 +2,44 @@
 [[links]]
 = Links
 
-// [[internal-links]]
-// == Internal links
+[[cross-references]]
+== Cross-references
 
-// [[external-links]]
-// == External links
+Follow these guidelines when adding cross-references within your documentation:
+
+* Include cross-references only when necessary.
+* If the information is critical, consider including it instead of cross-referencing.
+
+.Example AsciiDoc: Cross-reference
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+[[external-links]]
+== External links
+
+Follow these guidelines when linking externally:
+
+* Avoid links to external sites when possible. External links can change or be unreliable.
+* Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. See the _IBM Style Guide_ for information about using addresses in examples.
+* Default to followable, crawlable links. Do not use "nofollow" links unless absolutely necessary.
```

Comment:
These are relatively common HTML hyperlink concepts that have been around for awhile and are easy to find more info about (as you discovered!). I guess it depends on the level of HTML knowledge we're assuming about the writers using the SSG. Is it typical that the average tech writer working in a docs-as-code, Asciidoc workflow would know these concepts? My view is "yes." Maybe other people have a different view...

Reply:
I'm not familiar with those terms either, and would appreciate an explanation in the guide.

Reply:
This line is from the original guidance, which was for a much broader set of situations than docs. But I think you've got a point, Sharon and Breda, they're probably unknown to tech writers.
So maybe this line is not applicable to the documentation case really because we're being prescriptive about what to do in the rest of this guidance and writers are probably not in a position to use such types of links anyway? If that's the case, we could remove it.

Reply:
So I just checked and apparently it is possible to define a link in AsciiDoc with a nofollow option. Would look something like this:

```
link:www.example.com[My example link, opts=nofollow]
```
So I think that this guidance is valuable to keep. I personally didn't know much about the difference of the two, other than knowing that you have to do something special to make a link "nofollow". 

However, I don't think we need to go too deep into explanations here. Maybe we could make it clearer that the default is followable/crawlable, and you'd have to do something special to enable "nofollow", and to not do that :)

What about something like this:

```suggestion
* By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
```

Reply:
Your suggestion LGTM @bergerhoffer!

Reply:
Thanks @bredamc, I pushed the change (so that if @bburt-rh and @sbmetz agree, this can be approved if otherwise ready).

Reply:
This makes sense!

Reply:
That works for me! Thanks!

Reply:
Thanks all! @sbmetz would you mind approving the PR officially if it looks okay to you? (You go to files changed tab -> review changes -> hit the 'approve' radio button)